cpp/src/layout/barnes_hut.hpp

```diff
@@ -151,12 +151,14 @@ void barnes_hut(raft::handle_t const &handle,
   swinging   = d_swinging.data();
   traction   = d_traction.data();
 
+  thrust::fill(rmm::exec_policy(stream), d_old_forces.begin(), d_old_forces.end(), 0.f);
```

Comment:
Thanks Paul. I think we should add it in the exact version also

Reply:
Updated!